libcloud/compute/drivers/runabove.py

```diff
@@ -52,13 +39,20 @@ class RunAboveNodeDriver(NodeDriver):
     api_name = 'runabove'
 
     NODE_STATE_MAP = OpenStackNodeDriver.NODE_STATE_MAP
+    VOLUME_STATE_MAP = OpenStackNodeDriver.VOLUME_STATE_MAP
```

Comment:
RunAbove is based on OpenStack but they propose their own API, which is more simple than the OpenStack one. It has an abstraction layer which notably make easy to deal between datacenters.